### ISSUE TITLE
mirror the dropbox folder locally?

### ISSUE THREAD

**Issue by aristosv** [andreafabrizi/Dropbox-Uploader#343]
Hi,
Is there a way to mirror the dropbox folder locally?
So if I add or remove files from the dropbox folder, the changes would be reflected on the Linux computer?
Thanks

**Comment by tomwaldnz** [comment]
The official Dropbox client does that.

**Comment by trevorjwood** [comment]
Yes, but the official Dropbox client doesn't run on Unix/Linux - hence this project

**Comment by aristosv** [comment]
Yeah and I needed this for raspbian, so...I guess arm is not supported.

**Comment by tomwaldnz** [comment]
Dropbox does have a [Linux client](https://www.dropbox.com/install-linux).

**Comment by trevorjwood** [comment]
Ah, excellent. Haden't seen this. I'll give it a try tomorrow and see if it works on raspbian

**Comment by trevorjwood** [comment]
OK, given it a try and it won't install - comes up with "unspecified error". I seem to remember seeing somewhere that it doesn't work because of the type of processor the PI uses :-(
